firefox_ui_tests/remote/security/test_dv_certificate.py

```diff
@@ -0,0 +1,103 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import By, Wait
+
+from firefox_ui_harness.decorators import skip_under_xvfb
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+class TestDVCertificate(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+        self.test_url = "https://ssl-dv.mozqa.com"
+        self.cert = None
+        self.locationbar = self.browser.navbar.locationbar
+        self.identity_popup = self.browser.navbar.locationbar.identity_popup
+
+    def tearDown(self):
+        self.marionette.set_context("chrome")
+        self.identity_popup.close(force=True)
+        FirefoxTestCase.tearDown(self)
+
+    @skip_under_xvfb
+    def test_larry_blue(self):
+
+        with self.marionette.using_context('content'):
+            self.marionette.navigate(self.test_url)
+
+        self.cert = self.browser.tabbar.selected_tab.certificate
+
+        favicon_hidden = self.marionette.execute_script("""
+          return arguments[0].hasAttribute("hidden");
+        """, script_args=[self.locationbar.favicon])
+
+        self.assertFalse(favicon_hidden)
+
+        self.assertIn('identity-icons-https',
+                      self.locationbar.favicon.value_of_css_property('list-style-image'),
+                      'There is a lock icon')
+
+        self.assertEqual(self.identity_popup.box.get_attribute('className'),
+                         'verifiedDomain',
+                         'Identity is verified')
+
+        panel = self.identity_popup.popup
+        self.identity_popup.box.click()
+        self.wait_for_condition(lambda _: self.identity_popup.is_open)
+
+        doorhanger_class = self.identity_popup.popup.get_attribute('className')
```

Comment:
Put this directly into assertEqual. There is no value to define an extra variable here.